src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/service/KalendarService.java

```diff
@@ -25,6 +26,16 @@
 @Service
 public class KalendarService {
 
+    private static final String USER_NOT_FOUND_TOKEN = "User not found for clientToken=";
```

Comment:
I would not put exception messages as constants if we are not reusing them anywhere else